engine/runtime-compiler/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -900,7 +863,9 @@ class Compiler(
   ): Unit = {
     if (config.isStrictErrors) {
       val diagnostics = modules.flatMap { module =>
-        val errors = gatherDiagnostics(module)
+        val errors =
+          if (context.wasLoadedFromCache(module)) List()
```

Comment:
So where are the errors reported if the module was loaded from cache?

Reply:
No, the module cannot even be stored into the cache, if it contains this error.